VideoLocker/src/main/java/org/edx/mobile/module/db/IDatabase.java

```diff
@@ -345,8 +343,7 @@ public Integer updateAsDownloadingByVideoId(VideoModel model,
      * @param callback
      */
     public List<VideoModel> getDownloadedVideoListForCourse(String courseId,
-                                                            DataCallback<List<VideoModel>>
-                                                                    callback);
+                                                            DataCallback<List<VideoModel>> callback);
```

Comment:
I think this is a little over the line character limit, so the indentation should be reverted to one tab.